pkg/controller/gardenletdeployer/actuator.go

```diff
@@ -445,10 +445,12 @@ func (a *Actuator) reconcileSeedSecrets(ctx context.Context, obj client.Object,
 	if spec.Backup == nil {
 		return nil
 	}
+	if spec.Backup.CredentialsRef.APIVersion != "v1" || spec.Backup.CredentialsRef.Kind != "Secret" {
```

Comment:
Nit:
```suggestion
	if spec.Backup.CredentialsRef.APIVersion != corev1.SchemeGroupVersion.String() || spec.Backup.CredentialsRef.Kind != "Secret" {
```

Same for other occurences